client/package.json

```diff
@@ -5,6 +5,7 @@
     "normalize.css": "5.0.0",
     "preact": "^7.1.0",
     "preact-compat": "^3.9.2",
+    "react-markdown": "^2.4.4",
```

Comment:
I cannot find where it is used, does it need to be added as dependency ?

(ReactMarkdown is imported in `client/app/components/AccountConnection.jsx` but it seems to be never used, we may remove it I think)